model/llm/llm.go

```diff
@@ -123,14 +129,14 @@ var llmCodeRepairSourceFilePromptTemplate = template.Must(template.New("model-ll
 	- {{.}}{{ end }}
 `)))
 
-// llmCodeRepairSourceFilePrompt returns the prompt to code repair a source file.
-func llmCodeRepairSourceFilePrompt(data *llmCodeRepairSourceFilePromptContext) (message string, err error) {
+// Format returns the prompt to code repair a source file.
+func (ctx *llmCodeRepairSourceFilePromptContext) Format() (message string, err error) {
```

Comment:
Nitpick: Wouldn't make more sense calling this `Prompt`?

Reply:
I think it is just a context for a template. So I guess format makes sense. Prompt is again rather specific...